common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -110,6 +110,7 @@ jobs:
       maxParallel: 3
```

Comment:
do we really need to run for node 12.22 and another for 12.x?

Reply:
fixed.

Reply:
sorry, another thing, does it make sense to run regression for node18, if thats the main ver we use in our ci? 
imo wastes another 30 mins or so, which maybe is neglibile

Reply:
I still think it's important to test both the first LTS version of a major (18.12) and the latest one.  In this PR I switched around our regular ci build to use the latest, 18.x, and the regression tests to check the first Major.minor that we support.

Reply:
> We have run into compatibility issues previously due to this.

issues in the same minor? thats very concerning...

Reply:
i guess the appui issue from above is an example. resolving this comment